### PR TITLE
🌐 [i18n-KO] Translated `logging.md` to Korean

### PR DESCRIPTION
**Pull request by chhaewxn**
# What does this PR do?

Translated the `logging.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe, @nuatmochoi, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/main_classes/logging.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Python에는 종종 함께 사용되는 두 가지 로깅 시스템이 있습니다. 위에서 설명한 `logging`과 `warnings`입니다. `warnings`는 특정 범주로 경고를 세분화할 수 있습니다. 예를 들어, 이미 더 이상 사용되지 않는 기능이나 경로에 대해 `FutureWarning`이 사용되고, 곧 사용 중단될 기능을 알리기 위해 `DeprecationWarning`이 사용됩니다.
+
+트랜스포머 라이브러리에서는 두 시스템 모두를 사용합니다. `logging`의 `captureWarnings` 메서드를 활용하고 이를 조정하여 위에서 설명한 출력 수준 설정자들을 통해 이러한 경고 메시지들을 관리할 수 있도록 합니다.
+
+라이브러리 개발자에게는 다음과 같은 지침을 따르는 것이 좋습니다.
+
+- `warnings`는 라이브러리 개발자 및 `transformers`에 의존하는 라이브러리에서 우선적으로 사용해야 합니다.
+- `logging`은 라이브러리를 매일 사용하는 최종 사용자에게 사용해야 합니다.
+
+아래에서 `captureWarnings` 메서드에 대한 참고 사항을 확인할 수 있습니다.
```
Comment (fabxoe) [inline docs/source/ko/main_classes/logging.md:76]:
```suggestion
아래에서 `captureWarnings` 메소드에 대한 참고 사항을 확인할 수 있습니다.
```
이 부분은 glossary를 따라서 `메소드`로 바꿨어요. 개인적으로 오히려 `메서드`가 다른 책이나 튜토리얼에서 충분히 많이 쓰이고 있어서 사람들에게 익숙하고 혼용해도 문제가 없다고 생각을 하는데, 팍팍 하지만 일단 통일감을 위해서 바꿨어요. :)

**Review by fabxoe — CHANGES_REQUESTED** [review]
fix: suggestions

File: docs/source/ko/main_classes/logging.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+from transformers.utils import logging
+
+logging.set_verbosity_info()
+logger = logging.get_logger("transformers")
+logger.info("INFO")
+logger.warning("WARN")
+```
+
+
+이 로깅 모듈의 모든 메서드는 아래에 문서화되어 있으며, 주요 메서드는 현재 로거의 출력 수준을 가져오는 [logging.get_verbosity]와 원하는 출력 수준으로 설정하는 [logging.set_verbosity]입니다. 출력 수준은 (가장 적은 출력에서 가장 많은 출력 순으로) 다음과 같으며, 해당 수준에 대응하는 정수 값은 괄호 안에 표시됩니다.
```
Comment (jun048098) [inline docs/source/ko/main_classes/logging.md:55]:
```suggestion
이 로깅 모듈의 모든 메서드는 아래에 문서화되어 있으며, 주요 메서드는 현재 로거의 출력 수준을 가져오는 [logging.get_verbosity]와 원하는 출력 수준으로 설정하는 [logging.set_verbosity]입니다. 출력 수준은 (가장 적은 출력에서 가장 많은 출력 순으로) 다음과 같으며, 해당 수준에 대응하는 정숫값은 괄호 안에 표시됩니다.
```

File: docs/source/ko/main_classes/logging.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+- `transformers.logging.CRITICAL` 또는 `transformers.logging.FATAL` (정수 값, 50): 가장 심각한 오류만 보고합니다.
```
Comment (jun048098) [inline docs/source/ko/main_classes/logging.md:57]:
```suggestion
- `transformers.logging.CRITICAL` 또는 `transformers.logging.FATAL` (정숫값, 50): 가장 심각한 오류만 보고합니다.
```

File: docs/source/ko/main_classes/logging.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+- `transformers.logging.ERROR` (정수 값, 40): 오류만 보고합니다.
```
Comment (jun048098) [inline docs/source/ko/main_classes/logging.md:58]:
```suggestion
- `transformers.logging.ERROR` (정숫값, 40): 오류만 보고합니다.
```

File: docs/source/ko/main_classes/logging.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+- `transformers.logging.WARNING` 또는 `transformers.logging.WARN` (정수 값, 30): 오류와 경고만 보고합니다. 이는 라이브러리에서 기본으로 사용되는 수준입니다.
```
Comment (jun048098) [inline docs/source/ko/main_classes/logging.md:59]:
```suggestion
- `transformers.logging.WARNING` 또는 `transformers.logging.WARN` (정숫값, 30): 오류와 경고만 보고합니다. 이는 라이브러리에서 기본으로 사용되는 수준입니다.
```

File: docs/source/ko/main_classes/logging.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+- `transformers.logging.INFO` (정수 값, 20): 오류, 경고, 그리고 기본적인 정보를 보고합니다.
```
Comment (jun048098) [inline docs/source/ko/main_classes/logging.md:60]:
```suggestion
- `transformers.logging.INFO` (정숫값, 20): 오류, 경고, 그리고 기본적인 정보를 보고합니다.
```

File: docs/source/ko/main_classes/logging.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+- `transformers.logging.DEBUG` (정수 값, 10): 모든 정보를 보고합니다.
```
Comment (jun048098) [inline docs/source/ko/main_classes/logging.md:61]:
```suggestion
- `transformers.logging.DEBUG` (정숫값, 10): 모든 정보를 보고합니다.
```

**Review by jun048098 — COMMENTED** [review]
fix: suggestions

File: docs/source/ko/main_classes/logging.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+기본적으로 모델 다운로드 중에는 `tqdm` 진행 표시줄이 표시됩니다. [`logging.disable_progress_bar`]와 [`logging.enable_progress_bar`]를 사용하여 이 동작을 숨기거나 다시 표시할 수 있습니다.
+
+## `logging` vs `warnings`[[transformers.utils.logging.captureWarnings]]
+
+Python에는 종종 함께 사용되는 두 가지 로깅 시스템이 있습니다. 위에서 설명한 `logging`과 `warnings`입니다. `warnings`는 특정 범주로 경고를 세분화할 수 있습니다. 예를 들어, 이미 더 이상 사용되지 않는 기능이나 경로에 대해 `FutureWarning`이 사용되고, 곧 사용 중단될 기능을 알리기 위해 `DeprecationWarning`이 사용됩니다.
+
+트랜스포머 라이브러리에서는 두 시스템 모두를 사용합니다. `logging`의 `captureWarnings` 메서드를 활용하고 이를 조정하여 위에서 설명한 출력 수준 설정자들을 통해 이러한 경고 메시지들을 관리할 수 있도록 합니다.
+
+라이브러리 개발자에게는 다음과 같은 지침을 따르는 것이 좋습니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/logging.md:71]:
```suggestion
라이브러리 개발자는 다음과 같은 지침을 따르는 것이 좋습니다.
```

File: docs/source/ko/main_classes/logging.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+- `warnings`는 라이브러리 개발자와 `transformers`에 의존하는 라이브러리 개발자들에게 유리할 것입니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/logging.md:73]:
```suggestion
- `warnings`는 라이브러리 개발자와 `transformers`에 의존하는 라이브러리 개발자들에게 유리합니다.
```
더 자연스러운 것 같아 표현을 바꿔봤어요! :)

File: docs/source/ko/main_classes/logging.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+- `logging`은 평범한 프로젝트에서 라이브러리를 개발하기보다는 라이브러리를 사용하는 최종 사용자들에게 유리할 것입니다.
```
Comment (ahnjj) [inline docs/source/ko/main_classes/logging.md:74]:
```suggestion
- `logging`은 일반적인 프로젝트 라이브러리 개발자보다는, 라이브러리를 사용하는 최종 사용자들에게 유리할 것입니다.
```
더 이해가 될것 같은 표현으로 개발자를 붙여봤어요!

**Review by ahnjj — CHANGES_REQUESTED** [review]

**Review by stevhliu — APPROVED** [review]
Thanks! 🤗

**Pull request merged**